app/pages/Pdf/ServicePdfPage.tsx

```diff
@@ -45,9 +50,23 @@ export const ServicePdfPage = () => {
 
   useEffect(() => {
     if (mapImgSrc !== null && !pdfSource) {
+      // The below lines compress the CSS contained in the style element so as to decrease
+      // the character count that is sent to the Translation API
+      // https://dev.to/derder56/how-to-build-a-css-minifier-with-8-lines-of-javascript-4bj3
+      const styleElement = styleRef.current;
+      if (styleElement) {
+        styleElement.innerHTML = styleElement.innerHTML
+          .replace(/([^0-9a-zA-Z.#])\s+/g, "$1")
+          .replace(/;}/g, "}")
+          .replace(/\/\*.*?\*\//g, "");
+      }
+
       // Map image has been fetched. We're now ready to make our request to the PDF conversion endpoint
       fetch(`/api/services/html_to_pdf`, {
-        body: JSON.stringify({ html: ref.current?.outerHTML }),
+        body: JSON.stringify({
+          html: ref.current?.outerHTML,
+          target_language: targetLangCode ?? null,
```

Comment:
I think this `?? null` doesn't do anything, since it looks like [`URLSearchParams.get()`](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get) will already return `null` if the parameter doesn't exist. `?? null` would also catch `undefined` and turn it into a `null`, but it doesn't actually look like it's possible for it to be `undefined` in the first place.